pkg/pods/manifest.go

```diff
@@ -111,6 +113,9 @@ func (manifest *Manifest) Write(out io.Writer) error {
 	return nil
 }
 
+// Bytes() always returns the bytes from the original manifest file that was
```

Comment:
This seems like a very significant API change!  Before this PR, `Bytes()` on an unsigned manifest returned the current state of the manifest. After this PR, `Bytes()` on an unsigned manifest _might_ return an older version of the struct.

To make it clear that it has such different behavior, can you rename this to `OriginalBytes()` and audit all the references to this method to clarify whether they should be updated to the new name or switched to `Marshal()`?